src/Components/v2/ArtworkFilter/ArtworkFilterMobileActionSheet.tsx

```diff
@@ -1,36 +1,56 @@
-import { Box, Button, Sans, Spacer } from "@artsy/palette"
-import { MobileTopBar } from "Components/MobileTopBar"
+import { Box, Button, Sans, Flex, color } from "@artsy/palette"
 import React, { SFC } from "react"
 import styled from "styled-components"
-import { useArtworkFilterContext } from "./ArtworkFilterContext"
+import {
+  useArtworkFilterContext,
+  initialArtworkFilterState,
+} from "./ArtworkFilterContext"
+import { isEqual, omit } from "lodash"
 
 export const ArtworkFilterMobileActionSheet: SFC<{
   children: JSX.Element
   onClose: () => void
 }> = ({ children, onClose }) => {
   const filterContext = useArtworkFilterContext()
 
+  // TODO: `filterContext.hasFilters` seems to be what we want but is never actually `false`?
+  const isReset = isEqual(
+    omit(filterContext.filters, "reset", "keyword"),
+    initialArtworkFilterState
+  )
```

Comment:
This is essentially accurate — the reset method resets to the `initialAuctionResultsFilterState` with the addition of `reset: true`, the keyword is omitted because it's not reflected in this UI, but resetting also removes the keyword.